kod/object/active/holder/nomoveon/battler/player/user.kod

```diff
@@ -5319,6 +5358,7 @@ messages:
 
       if Send(self,@CheckPlayerFlag,#flag=PFLAG_NO_MAGIC)
          AND NOT IsClass(oSpell,&Phase)
+         OR Send(self,@IsInCannotInteractMode)
```

Comment:
This should be:

     if (Send(self,@CheckPlayerFlag,#flag=PFLAG_NO_MAGIC)
          OR Send(self,@IsInCannotInteractMode))  
       AND NOT IsClass(oSpell,&Phase)

otherwise players are prevented from casting Phase again while phased